FirebaseAppCheck/Tests/Unit/Swift/AppCheckTests.swift

```diff
@@ -1,52 +0,0 @@
-/*
- * Copyright 2020 Google LLC
```

Comment:
It should be ok to keep this file, since unit tests may be deeper, but less comprehensive than API tests

Reply:
I believe Maksym set this up to test how the APIs work in Swift. It doesnt contain any unit tests that run. So is it ok to be removed since the new file expands this file's intention?

Reply:
I see. LGTM